defog/__init__.py

```diff
@@ -453,7 +456,18 @@ def generate_mysql_schema(
         cur = conn.cursor()
         schemas = {}
 
-        print("Getting schema for each table in your database...")
+        if len(tables) == 0:
+            # get all tables
+            table_schema = self.db_creds.get("database", "")
+            cur.execute(
+                "SELECT table_name FROM information_schema.tables WHERE table_schema = '{table_schema}';"
```

Comment:
just checking if you plan to format the `table_schema` variable within the query string? If so, I think we might be missing an `f` before the start of the double quotes 🤔

Reply:
Thanks for finding this – fixed!